app/assets/stylesheets/styles/_events.scss

```diff
@@ -6,47 +6,67 @@
   box-sizing: border-box;
   min-height: 15em;
   overflow: auto;
-
-  @media (min-width:$break-large) {
-    // width: calc(100% - 30em);
-    // float: right;
-
-  }
 }
 
 .card-front, .card-back {
 
-
-
-
   .card-button {
     margin-right: $margin;
   }
 
   .card-image {
   margin: auto auto;
-  width: 90%;
+  max-width: 90%;
+  display: block;
+  max-height: 15em;
+  margin-bottom: 1em;
   }
 
   .left, .right {
     min-height: calc(15em + (2 * #{$padding}));
     padding: $card-padding;
     box-sizing: border-box;
+    width: 100%;
+    overflow-y: auto;
   }
 
   .left {
-    width: 35%;
-    float: left;
-    clear: left;
-    border-right: $divider;
+    border-bottom: $divider;
   }
 
   .right {
-    width: 60%;
-    float: right;
-    clear: right;
+
+  }
+
+  @media (min-width: 62.19em) {
```

Comment:
Just curious where these measurements are coming from. 62.19em == 995.04px here I think. Is that intentional?